src/devices/Bmxx80/Bmxx80Base.cs

```diff
@@ -49,15 +49,19 @@ public enum CommunicationProtocol
         }
 
         /// <summary>
-        /// The variable _temperatureFine carries a fine resolution temperature value over to the
+        /// The variable TemperatureFine carries a fine resolution temperature value over to the
         /// pressure compensation formula and could be implemented as a global variable.
         /// </summary>
-        protected int TemperatureFine;
+        protected int TemperatureFine
+        {
+            get;
+            set;
+        }
 
         /// <summary>
         /// The temperature calibration factor.
         /// </summary>
-        protected virtual int _tempCalibrationFactor => 1;
+        protected virtual int TempCalibrationFactor => 1;
```

Comment:
This is source breaking change, but I doubt there are a lot of people subclassing outside of the repo so I'm fine with this fix.

Reply:
I'd still go with this fix since it's very unlikely to break anyone and if it is well... they should have contributed it back :smiling_imp:

Reply:
I saw that and thought the same. If I remember correctly, there where only like 1 or 2 occassions of fixes like this I had to do.

Reply:
right, and I believe it is the right thing to fix them (at least the ones that I've spotted) so it's fine to take these breaking changes.